src/qmapshack/gis/trk/CGisItemTrk.cpp

```diff
@@ -506,6 +506,17 @@ QString CGisItemTrk::getInfo(quint32 feature) const
             IUnit::self().meter2elevation(totalDescent, val2, unit2);
 
             str += tr(", %1%2%3, %4%5%6").arg(QChar(0x2197)).arg(val1).arg(unit1).arg(QChar(0x2198)).arg(val2).arg(unit2);
+
+            if(actCnt == 1 && acts.contains(CTrackData::trkpt_t::eAct20Foot))
```

Comment:
This shouldn't be limited to tracks with a single activity. There is also a track statistic per activity.

Reply:
So how should I handle it so it is only calculated for the walking part?

Reply:
Add a method to `CActivityTrk` to get the `summary_t` structure of an activity.